src/components/header/styles.scss

```diff
@@ -1,18 +1,37 @@
 @import '~styles/settings';
 
 .c-header {
+  position: fixed;
   width: 100%;
-  height: 50px;
+  height: 55px;
   font-family: $font-family;
   font-size: 14px;
   display: flex;
   align-items: center;
   padding: 0 50px;
+  background: $color-white;
+  background: linear-gradient(180deg, $color-white 0%, transparent 100%);
+  z-index: $z-index-4;
 
-  .icon {
-    width: 20px;
-    max-height: 40px;
-    fill: $color-gray-1;
+  .menu-button {
+    height: 40px;
+    position: absolute;
+    font-size: 20px;
+    text-transform: uppercase;
+    display: flex;
+    align-items: center;
+    justify-content: center;
     cursor: pointer;
+
+    .menu-icon {
+      margin-right: 16px;
+      position: relative;
+      bottom: 2px;
```

Comment:
why do you need this? If the reason is to better center the icon, use a `transform`.